SPEC.md

```diff
@@ -2274,9 +2414,23 @@ Default unit is Bytes ("B").
 
 ## String sub(String, String, String)
 
+:pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
+
 Given 3 String parameters `input`, `pattern`, `replace`, this function will replace any occurrence matching `pattern` in `input` by `replace`.
 `pattern` is expected to be a [regular expression](https://en.wikipedia.org/wiki/Regular_expression). Details of regex evaluation will depend on the execution engine running the WDL.
 
+## Array[Int] range(Int)
```

Comment:
@geoffjentry does `Array[Int] range(Int)`, `Array[Array[X]] transpose(Array[Array[X]])`, `Pair(X,Y) zip(X,Y)`, and `Pair(X,Y) cross(X,Y)` exist in Cromwell?